docs/source/datasets.rst

```diff
@@ -33,7 +32,101 @@ to adhere to a certain format. ASReview accepts the following formats:
    be comma separated and UTF-8 encoded. For CSV files, the software accepts a set of 
    predetermined labels in line with the ones used in RIS files. 
 
-
+Metadata
+~~~~~~~~
+Each entry in the data set should hold metadata on a paper. Mandatory metadata are title and abstract. Other metadata such as id, author, date, and keywords are optional. When using ASReview in simulation mode, an additional binary variable to indicate labeling decisions is required, called included or label_included. All entries in the data set need to be screened, with the goal to select all relevant papers.
+
++----------+---------------------------------------------------------------------------------------------------------+-----------+
+| Name     | CSV names                                                                                               | Mandatory |
++==========+=========================================================================================================+===========+
+| title    | title, primary_title                                                                                    | yes       |
```

Comment:
Actually, either a title or an abstract is necessary. If it doesn't work when only one is present, it's a bug. (I should have fixed a bug that prevented this in the latest pull request.)

Reply:
This morning I tested both, either missing the titles or abstracts in a .csv file, but both had some issues. Missing the titles caused problems in the third step already, then it could not find any papers at all. When missing the abstracts, step 5 fails, however, when you close the project and open it again, it kind of works?

Reply:
I should have fixed these issues in the latest pull request that was merged just a few minutes ago. If it doesn't work still, let me know and I'll try to see what the problem is.

Reply:
Clarified by Sofie.